.code-workspace.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Sure you want this under git control?  Maybe belongs in .gitignore

Reply:
Good catch. I'll add it to .gitignore.